applications/StructuralMechanicsApplication/tests/test_StructuralMechanicsApplication.py

```diff
@@ -228,6 +235,9 @@ def AssembleTestSuites():
     smallSuite.addTests(KratosUnittest.TestLoader().loadTestsFromTestCases([TDynamicSchemesTests]))
     # Eigenvalues Postprocessing Process test
     smallSuite.addTests(KratosUnittest.TestLoader().loadTestsFromTestCases([TTestPostprocessEigenvaluesProcess]))
+    # Adjoint Elements
+    smallSuite.addTests(KratosUnittest.TestLoader().loadTestsFromTestCases([TTestCrBeamAdjointElement]))
```

Comment:
how long do these tests take?

Reply:
these two are tests of single elements, they take ~ 0.02 sec